deploy/docker-compose.yml

```diff
@@ -78,6 +78,7 @@ services:
       WAIT_HOSTS_TIMEOUT: 60
       CREDENTIAL_PUBKEY: b9679bbf526a1c936cd1144b56a370d376fa8246b248cd72f952b45a2f20bdad
       CREDENTIAL_PRIVKEY: 56a3c52cd039d9b73a1720052600a20962350b1ea169b4783cefbf87ed99406a
+      WHITELIST_VALIDATION: 1
```

Comment:
Please, move confirmation time to docker-compose.dev or docker-compose.ci

Reply:
Moved to dev